tests/system/test_system.py

```diff
@@ -662,6 +665,38 @@ def test_log_root_handler(self):
         self.assertEqual(len(entries), 1)
         self.assertEqual(entries[0].payload, expected_payload)
 
+    def test_log_handler_otel_integration(self):
```

Comment:
Is it possible to also add a system test without the otel sdk imported? 

I'm not exactly sure what that should look like, but I want to make sure we have coverage of the situation where otel isn't used at all

Reply:
Is it possible to create another file for the newly added system tests, so that the existing TCs aren't importing otel?

Reply:
Yeah, you can add an extra file. Or just import within the test functions. Or use a test class. Whatever seems cleanest

Does Otel behave any differently based on having the module installed? Or is import state the important part?

Reply:
I actually managed to resolve this by creating a decorator that deletes the otel SDK imports after the test case gets run. I don't think it's perfect but it feels good enough.

Reply:
Ok great! Do you have any test cases that test `_retrieve_current_open_telemetry_span` without otel included?